web/src/components/contact/ContactDialog.vue

```diff
@@ -40,6 +40,8 @@
           v-model="form.message"
           :label="$t('contact.message')"
           :error-messages="messageErrors"
+          clearable
+          hint="Details of your event (who, what, when, where)"
```

Comment:
Please avoid using raw text and add strings to en.json, so we can use vue-18n.

More details [here](http://kazupon.github.io/vue-i18n/guide/formatting.html).

Reply:
Also after reading through the [props](https://vuetifyjs.com/en/api/v-textarea/#props), I think "auto-grow" might be helpful here for long messages.